settings.gradle.kts

```diff
@@ -1,7 +1,98 @@
+/**
+ * settings.gradle.kts is used for bootstrapping a build.
+ *
+ * This is based on the xtc-application-tempalte repository, so understand how it works
+ * and how to supply credentials. If you have put working GitHub credentials in your
+ * $GRADLE_USER_HOME/gradle.properties already, this should just work.
+ */
+pluginManagement {
+    repositories {
+        maven {
+            val gitHubUrl: String by settings
+            val gitHubUser: String by settings
+            val gitHubToken: String by settings
```

Comment:
An attempt to execute this command (as suggested by the PR instructions)
`./gradlew build --scan --info --stacktrace | tee ~/build.log`
produces the following:
<
platform-new$ ./gradlew build --scan --info --stacktrace | tee ~/build.log
Initialized native services in: /Users/ggleyzer/.gradle/native
Initialized jansi services in: /Users/ggleyzer/.gradle/native
Received JVM installation metadata from '/Library/Java/JavaVirtualMachines/zulu-21.jdk/Contents/Home': {JAVA_HOME=/Library/Java/JavaVirtualMachines/zulu-21.jdk/Contents/Home, JAVA_VERSION=21.0.1, JAVA_VENDOR=Azul Systems, Inc., RUNTIME_NAME=OpenJDK Runtime Environment, RUNTIME_VERSION=21.0.1+12-LTS, VM_NAME=OpenJDK 64-Bit Server VM, VM_VERSION=21.0.1+12-LTS, VM_VENDOR=Azul Systems, Inc., OS_ARCH=aarch64}
Found daemon DaemonInfo{pid=55575, address=[8b868f3a-0929-46e4-adf7-914724ba630e port:63067, addresses:[/127.0.0.1]], state=Idle, lastBusy=1706039316882, context=DefaultDaemonContext[uid=85a8587f-ec9a-48fd-a0fc-cdabb309523c,javaHome=/Library/Java/JavaVirtualMachines/zulu-20.jdk/Contents/Home,daemonRegistryDir=/Users/ggleyzer/.gradle/daemon,pid=55575,idleTimeout=10800000,priority=NORMAL,applyInstrumentationAgent=true,daemonOpts=--add-opens=java.base/java.util=ALL-UNNAMED,--add-opens=java.base/java.lang=ALL-UNNAMED,--add-opens=java.base/java.lang.invoke=ALL-UNNAMED,--add-opens=java.prefs/java.util.prefs=ALL-UNNAMED,--add-opens=java.base/java.nio.charset=ALL-UNNAMED,--add-opens=java.base/java.net=ALL-UNNAMED,--add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED,-XX:MaxMetaspaceSize=384m,-XX:+HeapDumpOnOutOfMemoryError,-Xms256m,-Xmx512m,-Dfile.encoding=UTF-8,-Duser.country=US,-Duser.language=en,-Duser.variant]} however its context does not match the desired criteria.
Java home is different.
Wanted: DefaultDaemonContext[uid=null,javaHome=/Library/Java/JavaVirtualMachines/zulu-21.jdk/Contents/Home,daemonRegistryDir=/Users/ggleyzer/.gradle/daemon,pid=55798,idleTimeout=null,priority=NORMAL,applyInstrumentationAgent=true,daemonOpts=--add-opens=java.base/java.util=ALL-UNNAMED,--add-opens=java.base/java.lang=ALL-UNNAMED,--add-opens=java.base/java.lang.invoke=ALL-UNNAMED,--add-opens=java.prefs/java.util.prefs=ALL-UNNAMED,--add-opens=java.base/java.nio.charset=ALL-UNNAMED,--add-opens=java.base/java.net=ALL-UNNAMED,--add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED,-XX:MaxMetaspaceSize=384m,-XX:+HeapDumpOnOutOfMemoryError,-Xms256m,-Xmx512m,-Dfile.encoding=UTF-8,-Duser.country=US,-Duser.language=en,-Duser.variant]
Actual: DefaultDaemonContext[uid=85a8587f-ec9a-48fd-a0fc-cdabb309523c,javaHome=/Library/Java/JavaVirtualMachines/zulu-20.jdk/Contents/Home,daemonRegistryDir=/Users/ggleyzer/.gradle/daemon,pid=55575,idleTimeout=10800000,priority=NORMAL,applyInstrumentationAgent=true,daemonOpts=--add-opens=java.base/java.util=ALL-UNNAMED,--add-opens=java.base/java.lang=ALL-UNNAMED,--add-opens=java.base/java.lang.invoke=ALL-UNNAMED,--add-opens=java.prefs/java.util.prefs=ALL-UNNAMED,--add-opens=java.base/java.nio.charset=ALL-UNNAMED,--add-opens=java.base/java.net=ALL-UNNAMED,--add-opens=java.base/java.util.concurrent.atomic=ALL-UNNAMED,-XX:MaxMetaspaceSize=384m,-XX:+HeapDumpOnOutOfMemoryError,-Xms256m,-Xmx512m,-Dfile.encoding=UTF-8,-Duser.country=US,-Duser.language=en,-Duser.variant]

  Looking for a different daemon...
The client will now receive all logging from the daemon (pid: 55644). The daemon log file: /Users/ggleyzer/.gradle/daemon/8.5/daemon-55644.out.log
Starting 12th build in daemon [uptime: 10 mins 19.94 secs, performance: 99%, GC rate: 0.00/s, heap usage: 4% of 512 MiB, non-heap usage: 28% of 384 MiB]
Using 10 worker leases.
Now considering [/Users/ggleyzer/Development/platform-new] as hierarchies to watch
Watching the file system is configured to be enabled if available
File system watching is active
Wrote: /var/folders/86/ycb_v0n17174nt_jwynfpfp00000gn/T/xtc-init6754275328009075708/xtc-repo.init.gradle.kts
init.d script: Applying xtc-repo.init.gradle.kts from /var/folders/86/ycb_v0n17174nt_jwynfpfp00000gn/T/xtc-init6754275328009075708/xtc-repo.init.gradle.kts
Starting Build

FAILURE: Build failed with an exception.

* Where:
Settings file '/Users/ggleyzer/Development/platform-new/settings.gradle.kts' line: 12

* What went wrong:
Cannot get non-null property 'gitHubUser' on settings 'platform-new' as it does not exist

* Try:
> Run with --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org.

* Exception is:
org.gradle.api.InvalidUserCodeException: Cannot get non-null property 'gitHubUser' on settings 'platform-new' as it does not exist
        at org.gradle.kotlin.dsl.NonNullDynamicPropertyDelegate.getValue(PropertyDelegate.kt:81)
        at Settings_gradle$1$1$1.execute$lambda$1(settings.gradle.kts:12)
        at Settings_gradle$1$1$1.access$execute$lambda$1(settings.gradle.kts:10)
        at Settings_gradle$1$1$1$1.execute(settings.gradle.kts:16)
        at Settings_gradle$1$1$1$1.execute(settings.gradle.kts:15)
        at org.gradle.api.internal.artifacts.repositories.AuthenticationSupporter.credentials(AuthenticationSupporter.java:84)

>

Reply:
I have repeatedly tried to explain why the init.d scripts were in place, and what they provided syntactic sugaring for. Since I got nowhere with that, you now need to set up credential to the GitHub Maven repository for XTC manually. The process is described at length in the xtc-app-template repository. I have updated the platform readme with links to those resources. 

I don't see a build scan link anywhere? This is really the main I want from bug reports ....